carma_record/scripts/carma_record.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+while true
+    do
+        # Get the regex of the excluded topics
+        excluded_topics="$(rosparam get exclude_regex)"
+        if [ ! -z "${excluded_topics}" ]
```

Comment:
Since this will not record unless at least one exclusion is supplied we need to make sure that is documented somewhere. Maybe where the actual exclusion list parameter is set as well as the top of this file.